### ISSUE TITLE
Question: About indicator-session in AUR

### ISSUE THREAD

**Issue by Firef0x** [alucryd/aur-alucryd#21]
Hello @alucryd ,

Appreciate your great job. Actually I'm new to Unity, Pantheon and LightDM. It may be hasty in adopting package [AUR - indicator-session](https://aur.archlinux.org/packages/indicator-session). Sorry that this is not an issue for this repo and I just don't want to mess up the comment field of AUR. :sweat_smile: Here is my questions:

1. I thought `indicator-session` could only be used in Unity desktop and LightDM Unity Greeter before. But could it be used in other programs or other LightDM greeters? It seems that Pantheon have to use `indicator-pantheon-greeter-bzr` as mentioned in [Pantheon - Indicator-session menus not working](https://wiki.archlinux.org/index.php/Pantheon#Indicator-session_menus_not_working).

2. Pantheon is created for Elementary OS distribution, which is derived from Ubuntu. I guess Pantheon is designed to run in GTK+ 3 with Ubuntu patches. So why should we prevent from depending on GTK+ 3 with Ubuntu patches (i.e. gtk3-ubuntu) ?

3. Is Pantheon going to strip off anything with Ubuntu patches? Does it mean that Pantheon developer decide to stick on indicators 12.10 branch and give up supporting the latest version? Then, could we just split the 12.10 branch into a separate package indicator-xxx1210 so that users could distinguish Pantheon packages from Unity packages?

Hope for your reply! :heart_eyes: 

Yours sincerely! :bow:

**Comment by alucryd** [comment]
Yeah, indicators can be used by the Pantheon greeter and wingpanel, their top panel. Actually both work with 12.10 indicators (via a compile-time option) and the newest ones which use gtk3-ubuntu. However, patching gtk3 is a complete heresy and Arch Linux will never agree to that, ever. We only provide vanilla packages, and I won't be able to include Pantheon in our repos if it depends on gtk3-ubuntu.

That also goes for AUR, packages should try to depend on unpatched packages, and they also should be built from an official tarball, not a snapshot. If you want the latest indicators, you can use the bzr version or create indicator-session-ubuntu for example, so people know they need the horrible gtk3-ubuntu on their systems and that they will break everything :)

**Comment by Firef0x** [comment]
Hello @alucryd ,

I think the bzr version isn't needed anymore, since the `indicator-session` project is not under active development now, and every single commit would be tagged and released. https://code.launchpad.net/~indicator-applet-developers/indicator-session/trunk.15.04 shows that every single commit is tagged and released and not just a snapshot. https://launchpad.net/ubuntu/vivid/+source/indicator-session shows that there are official tarballs for those releases.

Removing `indicator-session-bzr` and creating `indicator-session-ubuntu` instead would be a good idea. Actually, I have filed a request to merge `indicator-session-bzr` into `indicator-session` two weeks ago and planning that once it was done, I will upload the package `indicator-session-ubuntu`. But no response so far.

Excuse me, one more question: Why did you decide to remove the translation files for `indicator-session` in commit https://github.com/alucryd/aur-alucryd/commit/262d9bbdf64c58af489c602e8339a7fc12a58d65 . That's the only difference from `12.10.4-4` to `12.10.4-5` , and I am puzzled. :confused:

Hope for your reply! :heart_eyes: 

Yours sincerely! :bow:

**Comment by alucryd** [comment]
Nah, keeping the bzr package around is useful because you don't need to bump it every once in a while. BTW, no matter how you look at them, those are snapshots, not release tarballs. What kind of a release number is 12.10.5+15.04.20150327-0ubuntu1?

As for translations, I removed them because they're  too old. It was City-Busz who was hosting them keeping them up to date. He stopped caring a while ago, and as I care even less, I decided to drop them altogether. Doesn't change much anyway.
